lib/truffle/java.rb

```diff
@@ -0,0 +1,50 @@
+###### BEGIN LICENSE BLOCK ######
```

Comment:
that would mean `lib/java.rb` which seems odd and bad (since it means `lib` would be on `$LOAD_PATH` and stuff like `lib/mri/set.rb` would inadvertently work). Or do you mean `lib/java/java.rb` and `lib/java/java/array_java_proxy.rb` ?